mantaray/views.py

```diff
@@ -20,6 +20,7 @@
 class _UserList:
     def __init__(self, irc_widget: IrcWidget):
         self.treeview = ttk.Treeview(irc_widget, show="tree", selectmode="extended")
+        self.treeview.tag_configure("away", foreground="#95968c")
```

Comment:
Nick changes currently mark the user as not away, because it removes the user and adds again but with no tags. Can fix in separate PR.